tasks/request_status_for_job/request_status_for_job.py

```diff
@@ -2,25 +2,24 @@
 from typing import Dict, Any, List
 
 import database
+import requests_db
 from cumulus_logger import CumulusLogger
 from requests_db import DatabaseError
 
-ASYNC_OPERATION_ID_KEY = 'asyncOperationId'
-
-DATABASE_SCHEMA_NAME = 'orca'
+INPUT_JOB_ID_KEY = 'asyncOperationId'
 
 LOGGER = CumulusLogger()
 
 
-def task(async_operation_id: str, db_connect_info: Dict) -> Dict[str, Any]:
+def task(job_id: str, db_connect_info: Dict) -> Dict[str, Any]:
     f"""
 
     Args:
-        async_operation_id: The unique asyncOperationId of the recovery job.
+        job_id: The unique asyncOperationId of the recovery job.
         db_connect_info: The {database}.py defined db_connect_info.
     Returns:
         A dictionary with the following keys:
-            'asyncOperationId' (str): The {async_operation_id}.
+            'asyncOperationId' (str): The {job_id}.
             'job_status_totals' (Dict): A dictionary with the following keys:
                 'pending' (int)
                 'success' (int)
```

Comment:
I would add staged since that is an additional state talked about in the working groups.

Reply:
Alright. I'll update the docs.